Applications/ApplicationsLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -88,7 +88,7 @@ void UncoupledProcessesTimeLoop::setInitialConditions(
                 ode_sys.getMatrixSpecifications()));
 
         auto& x0 = *_process_solutions[pcs_idx];
-        pcs.setInitialConditions(x0);
+        pcs.setInitialConditions(t0, x0);
```

Comment:
In general simulations don't need to start at t=0. Furthermore the Parameter class which backs the ICs now requires a time argument.